backend/api/views/failed_asset_logs.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import response as drf_response
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from api.models import ComputeTask
+from api.views import utils as view_utils
+from substrapp.models import asset_failure_report
+from substrapp.models import FailedAssetKind
+
+
+class FailedAssetLogsViewSet(view_utils.PermissionMixin, viewsets.GenericViewSet):
+    queryset = asset_failure_report.AssetFailureReport.objects.all()
```

Comment:
is there a specific reason we want a `.all()` here? For caching? 
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#all

Reply:
Following DRF examples: https://www.django-rest-framework.org/api-guide/generic-views/#examples